content/en/lotus/manage/chain-management.md

```diff
@@ -235,3 +235,155 @@ It is possible to _prune_ the current chain data used by Lotus to reduce the nod
     ```
 
 1. That's it!
+
+## SplitStore
+
+Version 1.17.1 introduces SplitStore, an actively scalable blockstore for the Filecoin chain which reduces the performance impact of large blockstores.
+
+SplitStore is a freestanding compacting blockstore that allows you to keep a small (60-275GiB) working set in a hot blockstore and reliably archive out of scope objects in a coldstore. The coldstore can also be a discard store, whereby out of scope objects are discarded or a universal badger blockstore (the default), which can be periodically garbage collected according to configurable user retention policies.
+
+To enable the splitstore, edit `.lotus/config.toml` and add the following:
+
+```toml
+[Chainstore]
+  # type: bool
+  # env var: LOTUS_CHAINSTORE_ENABLESPLITSTORE
+  EnableSplitstore = true
+```
+If you intend to use the discard coldstore, your also need to add the following:
+```toml
+[Chainstore.Splitstore]
+  # ColdStoreType specifies the type of the coldstore.
+  # It can be "universal" (default) or "discard" for discarding cold blocks.
+  #
+  # type: string
+  # env var: LOTUS_CHAINSTORE_SPLITSTORE_COLDSTORETYPE
+  ColdStoreType = "discard"
+```
```

Comment:
````suggestion
```toml
[Chainstore]
  # type: bool
  # env var: LOTUS_CHAINSTORE_ENABLESPLITSTORE
  EnableSplitstore = true
```

If you intend to use the discard-store you also need to add the following:

```toml
[Chainstore.Splitstore]
  # ColdStoreType specifies the type of the coldstore.
  # It can be "universal" (default) or "discard" for discarding cold blocks.
  #
  # type: string
  # env var: LOTUS_CHAINSTORE_SPLITSTORE_COLDSTORETYPE
  ColdStoreType = "discard"
```
````